RosNet/DataModel/RosBag.cs

```diff
@@ -43,6 +45,53 @@ public override string ToString()
             return s;
         }
 
+        /// <summary>
+        /// Makes a dictionary of all the topics and fieldnames in rosbag
+        /// </summary>
+        /// <returns>Dictionary with topics and corresponding fieldnames</returns>
+        public Dictionary<string, List<string>> GetConnectionFields()
+        {
+            var fieldsByTopic = new Dictionary<string, List<string>>();
+            foreach (Connection conn in Connections.Values)
+            {
+                fieldsByTopic.Add(conn.Topic, new List<string>());
+                foreach (FieldValue field in conn.MessageDefinition)
+                {
+                    fieldsByTopic[conn.Topic].Add(field.Name);
+                }
+            }
+            return fieldsByTopic;
+        }
 
+        /// <summary>
+        /// Makes a dictionary of timestamps and fieldvalues of field with fieldname fieldName
+        /// </summary>
+        /// <returns>Dictionary with time and corresponding fieldvalue</returns>
+        public Dictionary<(uint,uint),FieldValue> GetTimeSeries(string topic, string fieldName)
```

Comment:
Stop! You are wasting a lot of time by not just merging https://github.com/RevolveNTNU/RosNet/pull/70, first, since you now have to change this code to fix merge-conflicts afterwards!

Reply:
#70 wasn't set up as a PR when we wrote this, but we can merge it and add in Time here now

Reply:
Hvorfor lager dere så sykt mange brancher? Bare push alt til master, ikke vits i dette sinnsyke review-systemet som gir dere så mange utgaver av samme kode.